<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Online Computer Store</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/online-shopping.svg</normaloff>../icons/online-shopping.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditSearchBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSearch">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>5</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/search.svg</normaloff>../icons/search.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>130</y>
      <width>1001</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>home</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>283</x>
        <y>250</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LAPTOPS / NOTEBOOKS BEST SELLERS:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>290</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>ITEM1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>463</x>
        <y>290</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>ITEM2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>643</x>
        <y>290</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>ITEM3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>291</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LAPTOPS / NOTEBOOKS SUGGESTED ITEMS:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_36">
      <property name="geometry">
       <rect>
        <x>281</x>
        <y>60</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>System1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_37">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>60</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>System2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_38">
      <property name="geometry">
       <rect>
        <x>651</x>
        <y>60</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>System3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>mac</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>windows</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>linux</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>pc builder</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_24">
      <property name="geometry">
       <rect>
        <x>129</x>
        <y>40</y>
        <width>221</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Build With Intel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Build With ARM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_29">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Pre-Built</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Business</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>221</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_26">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>40</y>
        <width>221</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Computing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gaming</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_30">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Build With Intel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_31">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Build With Intel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_32">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>160</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Build With ARM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_33">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>160</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Build With ARM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_34">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>210</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Pre-Built</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_35">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>210</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Pre-Built</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>components</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>CPUs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Motherboard</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Cards</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Case</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>200</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Power Supply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Storage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>360</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Cooler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_23">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>360</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Operating System</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>328</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonLogin">
       <property name="text">
        <string>Log In</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/user.svg</normaloff>../icons/user.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRegister">
       <property name="text">
        <string>Register</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/register.svg</normaloff>../icons/register.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCart">
       <property name="text">
        <string>Cart</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/cart.svg</normaloff>../icons/cart.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>862</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonHome">
       <property name="text">
        <string>Home</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/home.svg</normaloff>../icons/home.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMac">
       <property name="text">
        <string>Mac</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/mac.svg</normaloff>../icons/mac.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWindows">
       <property name="text">
        <string>Windows</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/windows.svg</normaloff>../icons/windows.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLinux">
       <property name="text">
        <string>Linux</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/linux.svg</normaloff>../icons/linux.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPCBuilder">
       <property name="text">
        <string>PC Builder</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/build-pc.svg</normaloff>../icons/build-pc.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonComponents">
       <property name="text">
        <string>Components</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/computer-parts.svg</normaloff>../icons/computer-parts.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDiscussion">
       <property name="text">
        <string>Discussion</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/discussion.svg</normaloff>../icons/discussion.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
